### DISCUSSION TITLE
Any way to add a specific class to the dropdown?

### DISCUSSION THREAD

**Discussion by seabasss** [orchidjs/tom-select#437] [General]
I find `copyClassesToDropdown`, but is there a way to only add one or more specific classes to the dropdown? Copying all of them is not what I'm after. I also know that I can add classes to items and options, but it would be great to add classes to the dropdown itself so it's easier to target.

Thank you!

**Comment by oyejorge** [comment]
Custom dropdown template: https://tom-select.js.org/docs/#render-templates

**Reply by seabasss** [reply]
Thanks, looks like I can use
```
dropdown: function() {
    return '<div class="my-class"></div>';
}
```
Thanks!
